ctapipe/image/geometry_converter_astri.py

```diff
@@ -12,49 +12,49 @@ def astri_transformation_map():
     img_map = np.full([8 * 7, 8 * 7], -1, dtype=int)
 
     # Map values
-    img_map[0*8:1*8, 2*8:3*8] = np.arange(64).reshape([8,8])[::-1,:] + 34 * 64
```

Comment:
I'm not sure this is actually easier to read, but I guess at least it stops pep8 from complaining

Reply:
In the "other recommendations" part of the PEP8 they actually say this is **not** recommended.

https://www.python.org/dev/peps/pep-0008/#other-recommendations

> If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies). 

Here `:` has lower priority than `*`. I assume some pep8 tools might not implement this correctly .... if this is the case for the tools we use here, we maybe wanna consider using different tools or to configure the tools we use differently?

Reply:
I was just using the pep8 command line tool.

Reply:
This whole file is a bit of a hack anyhow, and will break if we change camera geometries (which is why I like the more generic geometry_converter better anyhow, so in any case it will disappear at some point and we shouldn't worry too much.  So Im' ok with at least stopping pep8 from complaining.

Reply:
Oh I didn't know this ... anyway .. just wanted to mention one could write it like this .... as well:
```python

foo = [
    34, 35, 36,
    29, 30, 31, 32, 33,
    22, 23, 24, 25, 26, 27, 28,
    15, 16, 17, 18, 19, 20, 21,
    8, 9, 10, 11, 12, 13, 14,
    3, 4, 5, 6, 7]
bar = [
    0, 0, 0,
    1, 1, 1, 1, 1,
    2, 2, 2, 2, 2, 2, 2,
    3, 3, 3, 3, 3, 3, 3,
    4, 4, 4, 4, 4, 4, 4,
    5, 5, 5, 5, 5]
bar = [
    2, 3, 4,
    1, 2, 3, 4, 5,
    0, 1, 2, 3, 4, 5, 6,
    0, 1, 2, 3, 4, 5, 6,
    0, 1, 2, 3, 4, 5, 6,
    1, 2, 3, 4, 5
]
for a, b, c in zip(foo, bar, baz):
    img_map[
        a * 8:a+1 * 8,
        b * 8:b+1 * 8
    ] = np.arange(64).reshape([8, 8])[::-1, :] + c * 64
```